packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.component.tsx

```diff
@@ -71,7 +87,7 @@ const VitalsHeader: React.FC<VitalsHeaderProps> = ({ patientUuid }) => {
                 <Tag type="red">
                   <span className={styles.overdueIndicator}>
                     <Time />
-                    {t('overdue', 'Overdue')}
+                    {t('overdue', 'Overdue').concat(' : ', overdueVitalsTag)}
```

Comment:
```suggestion
                   {`${t('overdue', 'Overdue')}: ${overdueVitalsTag}`}
```

You might also want to add this property to the `overdueIndicator` class:

```scss
.overdueIndicator {
  // other styles...
  min-inline-size: max-content;
}

The `max-content` value ensures that the element will be as wide as its content, without wrapping.